### DISCUSSION TITLE
Is it safe to change TR_VERSION_* in CMakeLists.txt to obfuscate this info for trackers ?

### DISCUSSION THREAD

**Discussion by diligiant** [transmission/transmission#4463] [General]
Ideally, I'd like to be able to set the "client" string to anything I want, like one does on web front-ends.

**Comment by ckerr** [comment]
What does "safe" mean in this question? :smile_cat: 

It's not recommended, since spoofing another torrent client or a different version is likely to be breaking trackers' rules.

**Reply by diligiant** [reply]
😜 meaning the software doesn't use it to alter build or behavior.
Merry Christmas!

**Reply by ckerr** [reply]
Yes, the the version string doesn't change any other behavior other than the version number :+1:  Nothing gets enabled or disabled based on the version.

**Reply by ckerr** [reply]
:christmas_tree:
